README.md

```diff
@@ -164,23 +219,35 @@ Sec-Signature:
   public-key=<pk>
   sig=<signature>
   sign-request-data=<include, headers-only>
-  timestamp=<timestamp from `sig`'s signing data>
 ```
 
 
 The canonical CBOR data (verifiable by the signature) should be computable from a request, and so does not need to be sent over the wire from the browser. The `Signed-Headers` header, and the value of `sign-request-data` should be enough to re-construct it server side, robust to things like header re-ordering, etc.
 
 
+### Extension: Metadata
+
+In addition to attesting trust in a user, an issuer may want to provide a limited amount of metadata in the token (and forward it as part of the SRR) to provide limited additional information about the token.
+
+This small change opens up a new application for Privacy Passes: embedding small amounts of information along with the token and SRR. This increases the rate of cross-site information transfer somewhat, but introduces new use-cases for passes like marking bad clients, so _distrust_ can propagate across sites as well as trust. 
```

Comment:
Don't mention the distrust case specifically until the private metadata section, just mention new use-cases in the abstract.